treemap/app/src/main.js

```diff
@@ -124,19 +124,42 @@ class TreemapViewer {
     this.treemap;
     /*  eslint-enable no-unused-expressions */
 
-    this.createHeader();
+    const urlEl = dom.find('a.lh-header--url');
+    urlEl.textContent = this.documentUrl.toString();
+    urlEl.href = this.documentUrl.toString();
+
+    this.viewModeSelector = this.createViewModeSelector();
+    this.bundleSelector = this.createBundleSelector();
     this.toggleTable(window.innerWidth >= 600);
     this.initListeners();
     this.setSelector({type: 'group', value: 'scripts'});
     this.render();
   }
 
-  createHeader() {
-    const urlEl = dom.find('a.lh-header--url');
-    urlEl.textContent = this.documentUrl.toString();
-    urlEl.href = this.documentUrl.toString();
+  createViewModeSelector() {
+    const viewModeSelectorEl = dom.find('select.view-mode-selector');
+    viewModeSelectorEl.textContent = ''; // Clear just in case document was saved with Ctrl+S.
 
-    this.createBundleSelector();
+    viewModeSelectorEl.addEventListener('change', () => {
+      const index = Number(viewModeSelectorEl.value);
+      const viewMode = this.viewModes[index];
+      this.setViewMode(viewMode);
+      this.render();
+    });
+
+    return viewModeSelectorEl;
+  }
+
+  updateViewModeSelector() {
+    this.viewModeSelector.textContent = '';
+    for (const [i, viewMode] of this.viewModes.entries()) {
+      const optionEl = dom.createChildOf(this.viewModeSelector, 'option');
+      optionEl.value = String(i);
+      optionEl.textContent = `${viewMode.label} (${viewMode.subLabel})`;
```

Comment:
something about "All" was fine before when they were all visible at the same time. but .. now it feels non-descriptive of the view.

how about "Explore" or "Contents" or... something. I dunno I just kinda want something with more intrigue.

don't feel that strongly tho